test/MenuItem/AsciiArtItemTest.php

```diff
@@ -159,4 +141,67 @@ public function testGetRowsReturnsStaticAltItemWhenWidthIsTooSmall()
         
         self::assertSame(['my alt'], $item->getRows($menuStyle));
     }
+
+    public function testWithAsciiArtCentered() : void
+    {
+        $menuStyle = $this->createMock(MenuStyle::class);
+
+        $menuStyle
+            ->expects($this->any())
+            ->method('getContentWidth')
+            ->will($this->returnValue(30));
+
+        $art = <<<ART
+        _ __ _
+       / |..| \
+       \/ || \/
+        |_''_|
+      PHP SCHOOL
+LEARNING FOR ELEPHANTS
+ART;
+
+        $item = new AsciiArtItem($art, AsciiArtItem::POSITION_CENTER);
+        $this->assertEquals(
+            [
+                '            _ __ _',
+                '           / |..| \\',
+                '           \/ || \/',
+                "            |_''_|",
+                '          PHP SCHOOL',
+                '    LEARNING FOR ELEPHANTS'
+            ],
+            $item->getRows($menuStyle)
+        );
+    }
+
+    public function testWithRealAsciiArtCenteredWithWhiteSpaceAtTheEndOfEachLine() : void
```

Comment:
This is the test for your end whitespace fix

Reply:
Much sexier than the one I made eh :)